### PR TITLE
Fixes #250: API documentation 'more info' links broken.

### LINKED ISSUE #250: API documentation 'more info' links broken
If I click on the 'more info...' links in https://github.com/wildfly/wildfly-operator/blob/main/doc/apis.adoc, they 404

### PR DESCRIPTION
**Pull request by rhusar**

### CODE REVIEW

**Comment by rhusar** [comment]
Just rebased to get the tests running green after the previous fix :-)

File: doc/user-guide.adoc [hunk 1]
```diff
@@ -107,7 +107,7 @@ or configure the transaction subsystem to store transaction log in database with
 link:https://docs.wildfly.org/23/wildscribe/subsystem/transactions/index.html#attr-use-jdbc-store[use of JDBC store].
 ====
 
-A `volumeClaimTemplate` cna be specifed to configure `Resources` requirements to store WildFly standalone data directory.
+A `volumeClaimTemplate` cna be specified to configure `Resources` requirements to store WildFly standalone data directory.
```
Comment (yersan) [inline doc/user-guide.adoc:110]: We forgot this one "**cna** be specified", for the sake of completeness, would you mind fixing it with your PR?

**Review by yersan — APPROVED** [review]

File: doc/user-guide.adoc (hunk 1 above)
Comment (rhusar) [reply doc/user-guide.adoc]: Haha, nice one. Fixed.

Comment (yersan) [reply doc/user-guide.adoc]: Thank you, @rhusar !

**Comment by rhusar** [comment]
@yersan Bump!

**Comment by jmesnil** [comment]
@rhusar thanks!

**Pull request merged**